### PR TITLE
Added mousemoosz domain and sub-domains

### PR DESCRIPTION
**Pull request by VaibhavSys**
The status page is still in progress, and so is the minecraft server.  The website needs some changes because of rebranding and some things which will take time, but the domains are working.

### CODE REVIEW

File: domains/mousemoosz.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "MouseMoosz",
+    "email": "mousemoosz@protonmail.com"
+    "description": "For my website"
+  }
+  "record": {
+    "CNAME": "mousemoosz.netlify.app"
```
Comment (MaskDuck) [inline domains/mousemoosz.json:8]: sorry netlify doesnt work with this service =(

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/mousemoosz.json (hunk 1 above)
Comment (MouseMoosz) [reply domains/mousemoosz.json:8]: Oh, well are there no workarounds either?

Comment (MaskDuck) [reply domains/mousemoosz.json:8]: you can try railway or github pages.

all of them are great 👍

Comment (MouseMoosz) [reply domains/mousemoosz.json:8]: Alright, I will use github pages.

**Comment by MouseMoosz** [comment]
The website isn't there yet, but it willl be.

**Review by MaskDuck — DISMISSED** [review]
kay good

**Comment by MaskDuck** [comment]
if you ever need help then feel free create a github issue or join our discord server we wont be mad =)

**Review by MaskDuck — APPROVED** [review]
nice

**Comment by MouseMoosz** [comment]
esh, I didn't format my json correctly, Sorry and thank you!

**Pull request merged**